apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/tips/index.js

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
Could we rename that folder to `contextual-tips` to make it a little more descriptive?

Reply:
+1 

We'll likely end up having code for "NUX tips" in FSE plugin so being more specific already now here would be helpful.

<img width="217" alt="image" src="https://user-images.githubusercontent.com/87168/81589825-7ee38580-93c2-11ea-9756-6d3c36eb8b52.png">

Reply:
sounds  good